### PR TITLE
Install/update add-ons first in automation

### PR DESCRIPTION
**Pull request by psiinon**
Always install / update add-ons synchronously in cmdline / daemon modes before running any other command line args.
The GUI will still act as know, updating add-ons in the background.

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: zap/src/main/java/org/parosproxy/paros/extension/history/ExtensionHistory.java [hunk 1]
```diff
@@ -992,6 +992,8 @@ private void purgeTemporaryParents(SiteMap map, SiteNode node) {
     @Override
     public void destroy() {
         super.destroy();
-        this.getHistoryReferencesTable().persistColumnConfiguration();
+        if (View.isInitialised()) {
+            this.getHistoryReferencesTable().persistColumnConfiguration();
```
Comment (psiinon) [inline zap/src/main/java/org/parosproxy/paros/extension/history/ExtensionHistory.java:996]: OK, this is unrelated, but found while testing 😉

**Comment by psiinon** [comment]
Changed to WIP while investigating potential issues @thc202 has pointed out...

**Comment by psiinon** [comment]
Updated and no longer WIP

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**